root/etc/e-smith/events/actions/nethserver-nextcloud-conf

```diff
@@ -49,6 +49,9 @@ fi
 
 if [[ -n $database ]]; then
     # we still use mariadb55, this is wrong we need rh-mariadb105
+
+    # Make sure loglevel is set to 2 (see https://github.com/nextcloud/server/pull/27334)
+    OCC "config:system:set loglevel --value=2 --type=integer"
```

Comment:
This is the default log warning and normally after the upgrade it is written in config.php of nextcloud

https://gist.github.com/stephdl/afd6b203068930caf0cba9e2580a2d62